src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -767,6 +785,8 @@ namespace InternalHeatGains {
                             // Set the default value of MRTCalcType as 'ZoneAveraged'
                             People(Loop).MRTCalcType = ZoneAveraged;
 
+                            bool ThermalComfortRuns = People(Loop).Fanger || People(Loop).Pierce || People(Loop).KSU || People(Loop).CoolingEffectASH55 || People(Loop).AnkleDraftASH55;
```

Comment:
`ThermalComfortRuns` is a confusing name here, because it's only true for a subset of the thermal comfort models (all of them except AdaptiveASH55 and AdaptiveCEN15251). If I'm following correctly, the 5 in this bool all require some additional input fields. So maybe there's a better variable name to distinguish this from `UsingThermalComfort` higher up?

Reply:
This is fixed.